acoular/base.py

```diff
@@ -265,8 +263,7 @@ def result(self, num=1):
 
 @deprecated_alias({'numchannels': 'num_channels', 'numsamples': 'num_samples'}, read_only=True, removal_version='25.10')
 class InOut(SamplesGenerator, SpectraGenerator):
-    """Abstract base class for any signal processing block that receives data from any
-    :attr:`source` domain and returns signals in the same domain.
+    """Abstract base class receiving from a :attr:`source` and returning signals in the same domain.
```

Comment:
```suggestion
    """
    Abstract base class receiving from a :attr:`source` and returning signals in the same domain.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`